### PR TITLE
TG-57: Layout footer

### PR DESCRIPTION
**Pull request by martinlotufo93**
image

### CODE REVIEW

**Review by Facund0Fernandez — COMMENTED** [review]
Remember to use camelCase in CSS styles

File: src/Components/Shared/Footer/footer.module.css [hunk 1]
```diff
@@ -1,11 +1,49 @@
...
   display: flex;
   flex-direction: column;
-  align-items: center;
-  background-color: #000000;
+  background-color: #ffff;
+  width: 75%;
+  height: 12%;
+  margin: 0% 0% 0% 25%;
+}
+.greenline {
```
Comment (Gearyandres94) [inline src/Components/Shared/Footer/footer.module.css:9]: This class should be camel case.

File: src/Components/Shared/Footer/footer.module.css [hunk 2]
```diff
@@ -1,11 +1,49 @@
...
+  width: 80%;
+  border-top: 8px solid #aace9b;
+  border-radius: 5px;
+}
+.content {
+  display: flex;
+  flex-direction: row;
+}
+.footertext {
```
Comment (Gearyandres94) [inline src/Components/Shared/Footer/footer.module.css:18]: This also should be camel case.

File: src/Components/Shared/Footer/footer.module.css [hunk 3]
```diff
@@ -1,11 +1,49 @@
...
+  display: flex;
+  flex-direction: column;
+  float: right;
+  padding-top: 0.5%;
+  width: 70%;
+}
+.footertext p {
```
Comment (Gearyandres94) [inline src/Components/Shared/Footer/footer.module.css:25]: this also should be camel case.

File: src/Components/Shared/Footer/footer.module.css [hunk 4]
```diff
@@ -1,11 +1,49 @@
...
+  float: right;
+  margin: 0% 0% 0% 90%;
+}
+.p1 {
+  padding: 0% 0% 0% 42.5%;
+}
+.p2 {
+  padding: 0% 0% 0% 9%;
+}
+.rowlinks {
```
Comment (Gearyandres94) [inline src/Components/Shared/Footer/footer.module.css:37]: this also should be camel case.

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]

File: src/Components/Shared/Footer/footer.module.css [hunk 5]
```diff
@@ -1,11 +1,49 @@
...
+  background-color: #ffff;
+  padding-top: 1%;
+  width: 40%;
+}
+.rowLinks a {
+  padding: 3%;
 }
 
-.main {
+/*.main {
```
Comment (Facund0Fernandez) [inline src/Components/Shared/Footer/footer.module.css:46]: This comment should be uncommented or deleted.

**Review by Facund0Fernandez — APPROVED** [review]

File: src/Components/Shared/Footer/index.js [hunk 6]
```diff
@@ -1,58 +1,112 @@
...
-        <div>
-          <a href={'https://www.facebook.com/radiumrocket'} target={'_blank'} rel="noreferrer">
-            <img
-              className={styles.socialIcon}
-              src={`${process.env.PUBLIC_URL}/assets/images/facebook.svg`}
-            />
+      <div className={styles.greenLine}></div>
+      <div className={styles.content}>
+        <nav className={styles.rowLinks}>
+          <a href={'#'}>
```
Comment (Gearyandres94) [inline src/Components/Shared/Footer/index.js:9]: all anchors a should be replaced with router links

File: src/Components/Shared/Footer/index.js [hunk 7]
```diff
@@ -1,58 +1,112 @@
...
+              />
+            </svg>
           </a>
-          <a href={'https://twitter.com/radiumrocket'} target={'_blank'} rel="noreferrer">
-            <img
-              className={styles.socialIcon}
-              src={`${process.env.PUBLIC_URL}/assets/images/twitter.svg`}
-            />
+          <a href={'https://www.facebook.com/radiumrocket'}>
+            {' '}
```
Comment (Gearyandres94) [inline src/Components/Shared/Footer/index.js:47]: This should be deleted.

**Review by GuidoCerioni — CHANGES_REQUESTED** [review]
Hey Martin, i see you added the .svg files AND then wrote them again inside you component. You should do it one way, or the other, not both. We are doing it by adding the files and then importing them. Check out [Facu PR](https://github.com/BaSP-m2022/franco-trackgenix-app/pull/19/files) and how he is using .svg

File: src/Components/Shared/Footer/index.js [hunk 8]
```diff
@@ -1,58 +1,112 @@
...
+                fill="#373867"
+              />
+            </svg>
+          </a>
+          <a href={'https://twitter.com/radiumrocket'}>
```
Comment (Gearyandres94) [inline src/Components/Shared/Footer/index.js:32]: This should also be replaced.

File: src/Components/Shared/Footer/index.js [hunk 9]
```diff
@@ -1,58 +1,112 @@
...
+                fill="#373867"
+              />
+            </svg>
           </a>
-          <a href={'https://twitter.com/radiumrocket'} target={'_blank'} rel="noreferrer">
-            <img
-              className={styles.socialIcon}
-              src={`${process.env.PUBLIC_URL}/assets/images/twitter.svg`}
-            />
+          <a href={'https://www.facebook.com/radiumrocket'}>
```
Comment (Gearyandres94) [inline src/Components/Shared/Footer/index.js:46]: This should be replaced.

File: src/Components/Shared/Footer/index.js [hunk 10]
```diff
@@ -1,58 +1,112 @@
...
+                fill="#373867"
+              />
+            </svg>
           </a>
-          <a href={'https://www.instagram.com/radium.rocket/'} target={'_blank'} rel="noreferrer">
-            <img
-              className={styles.socialIcon}
-              src={`${process.env.PUBLIC_URL}/assets/images/instagram.svg`}
-            />
+          <a href={'https://www.instagram.com/radium.rocket/'}>
```
Comment (Gearyandres94) [inline src/Components/Shared/Footer/index.js:61]: Replaces this too.

File: src/Components/Shared/Footer/index.js [hunk 11]
```diff
@@ -1,58 +1,112 @@
...
+                fill="#373867"
+              />
+              <path
+                d="M26.5737 1.17858C20.4647 0.495811 14.15 0.495811 8.04099 1.17858C4.49522 1.57487 1.63273 4.36807 1.21586 7.93229C0.485338 14.1783 0.485338 20.4881 1.21586 26.7341C1.63273 30.2983 4.49522 33.0915 8.04099 33.4878C14.15 34.1705 20.4647 34.1705 26.5737 33.4878C30.1195 33.0915 32.982 30.2983 33.3989 26.7341C34.1294 20.4881 34.1294 14.1783 33.3989 7.93229C32.982 4.36807 30.1195 1.57487 26.5737 1.17858ZM8.33459 3.80561C14.2485 3.14465 20.3663 3.14465 26.2801 3.80561C28.6241 4.06759 30.5018 5.91728 30.7734 8.23936C31.48 14.2813 31.48 20.385 30.7734 26.427C30.5018 28.7491 28.6241 30.5988 26.2801 30.8607C20.3663 31.5217 14.2485 31.5217 8.3346 30.8607C5.9906 30.5988 4.11294 28.7491 3.84135 26.427C3.13469 20.385 3.13469 14.2813 3.84135 8.23936C4.11294 5.91728 5.9906 4.06759 8.33459 3.80561Z"
+                fill="#373867"
+              />
+            </svg>
           </a>
+          <a href={'#'}>
```
Comment (Gearyandres94) [inline src/Components/Shared/Footer/index.js:83]: This too needs to be replaced with link routes.

**Comment by GuidoCerioni** [comment]
Also, you should change <a>'s for <Link />. You could see that also in Facu's PR

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]
The svg should be as an icon or code, not both. Delete the files in public the component should still looking fine. And follow this to change the anchors with links: https://v5.reactrouter.com/web/api/Link

**Review by HernanCazeneuve — APPROVED** [review]

**Review by Gearyandres94 — APPROVED** [review]
Great Job!!

**Review by Facund0Fernandez — APPROVED** [review]

**Review by laura-brussa12 — APPROVED** [review]

**Review by GuidoCerioni — APPROVED** [review]
Nice!!

**Pull request merged**